rust/plugin-lib/src/lib.rs

```diff
@@ -146,7 +146,12 @@ pub trait Plugin {
                       changes: &ConfigTable);
 
     #[allow(unused_variables)]
-    fn language_changed(&mut self, view: &mut View<Self::Cache>, new_lang: LanguageId) {}
+    fn language_changed(
```

Comment:
similarly in the pluging lib, assuming we've already updated the view to have the new language, passing it to the function directly might be unnecessary? I can understand passing the old language (because we've overwritten it in the view, and so the plugin has no other way of getting it) but I'm not sure we need the new language. This is similar to how we handle when the path changes; we update the view with the new path, and then pass along the old path in case it might be helpful.